shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -220,57 +197,513 @@ - (CGRect)accessibilityFrame {
   auto result =
       CGRectMake(rect.x() / scale, rect.y() / scale, rect.width() / scale, rect.height() / scale);
-  return UIAccessibilityConvertFrameToScreenCoordinates(result, _bridge->view());
+  return UIAccessibilityConvertFrameToScreenCoordinates(result, [self bridge]->view());
 }
 
 #pragma mark - UIAccessibilityElement protocol
 
 - (id)accessibilityContainer {
-  if ([self hasChildren] || _uid == kRootNodeId) {
+  if ([self hasChildren] || [self uid] == kRootNodeId) {
     if (_container == nil)
-      _container = [[SemanticsObjectContainer alloc] initWithSemanticsObject:self bridge:_bridge];
+      _container =
+          [[SemanticsObjectContainer alloc] initWithSemanticsObject:self bridge:[self bridge]];
     return _container;
   }
-  NSAssert(_parent != nil, @"Illegal access to non-existent parent of root semantics node");
-  return [_parent accessibilityContainer];
+  NSAssert([self parent] != nil, @"Illegal access to non-existent parent of root semantics node");
+  return [[self parent] accessibilityContainer];
 }
 
 #pragma mark - UIAccessibilityAction overrides
 
 - (BOOL)accessibilityActivate {
-  if (!_node.HasAction(blink::SemanticsAction::kTap))
+  if (![self node].HasAction(blink::SemanticsAction::kTap))
     return NO;
-  _bridge->DispatchSemanticsAction(_uid, blink::SemanticsAction::kTap);
+  [self bridge]->DispatchSemanticsAction([self uid], blink::SemanticsAction::kTap);
   return YES;
 }
 
 - (void)accessibilityIncrement {
-  if (_node.HasAction(blink::SemanticsAction::kIncrease)) {
-    _node.value = _node.increasedValue;
-    _bridge->DispatchSemanticsAction(_uid, blink::SemanticsAction::kIncrease);
+  if ([self node].HasAction(blink::SemanticsAction::kIncrease)) {
+    [self node].value = [self node].increasedValue;
+    [self bridge]->DispatchSemanticsAction([self uid], blink::SemanticsAction::kIncrease);
   }
 }
 
 - (void)accessibilityDecrement {
-  if (_node.HasAction(blink::SemanticsAction::kDecrease)) {
-    _node.value = _node.decreasedValue;
-    _bridge->DispatchSemanticsAction(_uid, blink::SemanticsAction::kDecrease);
+  if ([self node].HasAction(blink::SemanticsAction::kDecrease)) {
+    [self node].value = [self node].decreasedValue;
+    [self bridge]->DispatchSemanticsAction([self uid], blink::SemanticsAction::kDecrease);
   }
 }
 
 - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
   blink::SemanticsAction action = GetSemanticsActionForScrollDirection(direction);
-  if (!_node.HasAction(action))
+  if (![self node].HasAction(action))
     return NO;
-  _bridge->DispatchSemanticsAction(_uid, action);
+  [self bridge]->DispatchSemanticsAction([self uid], action);
   return YES;
 }
 
 @end
 
+@interface InactiveTextPosition : UITextPosition
+
+@property(nonatomic, readonly) NSUInteger index;
+
++ (instancetype)positionWithIndex:(NSUInteger)index;
+- (instancetype)initWithIndex:(NSUInteger)index;
+
+@end
+
+@implementation InactiveTextPosition
```

Comment:
Please explicitly synthesize all properties. Though the compiler will synthesize these properties for you, specifying the same makes the intent clearer and makes it obvious that the properties should be collected in dealloc in case ARC is not being used.

Reply:
We do not use ARC, so all the properties implicitly synthesized and set by callers during the lifetime of the object will be leaked when this object is collected.

Reply:
I'll attempt to remove these classes (see @goderbauer's suggestion to share these with the FlutterTextInputPlugin).

Reply:
Done (removed these classes)